pages/linux/qm-showcmd.md

```diff
@@ -0,0 +1,16 @@
+# qm showcmd
+
+> Show command line which is used to start the VM (debug info).
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Show command line for a specific virtual machine:
+
+`qm showcmd {{vm_id}}`
```

Comment:
```suggestion
`qm showcmd "{{vm_id}}"`
```

Optional fix: if placeholder can contain special characters to POSIX-compatible shell it can be quoted.

Reply:
IDs are numbers

Reply:
What numbers are exactly allowed? I mean what range of numbers.